### ISSUE TITLE
List view for Claims - add/edit fields (2)

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va-mobile-app#8865]
## Description
As part of this story we want to: start updating the content in the list view for Claims

## Acceptance Criteria

-  [ ] Behind Feature flag
-  [ ] Update heading on page (Your [active/closed] claims, decision reviews, and appeals)
-  [ ] Update the title copy for claim type (remove date from title)
-  [ ] Align all content with the [copy doc](https://docs.google.com/spreadsheets/d/16waC665d7uNRQmM83SJIm9bTDyRePOuB6r9G_aQE8Tw/edit?gid=0#gid=0)
-  [ ] VQA with Lauren / Misty

## List view variations (Copied from [Figma](https://www.figma.com/design/kSlXLwTVLunM0fqCG95k5x/%F0%9F%94%8D-Claims-2.0---Working---VA-Mobile?node-id=1563-16994&t=GbeXxqLFv5xms2Ho-4))

<img width="212" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMHVDQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3b8b92cf05f798c8f36ad03344f8cd19d81be26e/image.png" alt="image.png" />

## Notes & Open Questions

- [ ] Does not include the "Step X of X" feature
- [ ] Does not include the "Add Moved to this step on [date]" feature
- [ ] Does not include the "More information needed" LabelTag

All of the above are blocked by [8822](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8822)

**Comment by ala-yna** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VA-Mobile-60f1a34998bc75000f2a489f/issues/department-of-veterans-affairs/va-mobile-app/8865?planning-poker) with Zenhub @alexandec @Sparowhawk @dumathane

**Comment by alexandec** [comment]
Lauren approved [on slack](https://adhoc.slack.com/archives/C067S0FRP0C/p1719511705327109)

**Comment by rbontrager** [comment]
@alexandec Appeals in the list view also need to be updated according to Lauren. There were no appeals on the page for the VQA so she missed it.  Also I don't know if you want to start updating the detox tests now or if we need to make another ticket to wait for when the feature is done.

**Comment by alexandec** [comment]
@rbontrager got it, I've made that appeals change. I also updated the unit tests and the E2E tests to account for the changes in this ticket

**Comment by rbontrager** [comment]
Verified that the changes from this ticket are behind a feature flag.  Verified that the heading for list view for claims/appeals has changed to Your [active/closed] claims, decision reviews, and appeals).  Verified that the date has been removed from the claim type in the list view details and that the received by date and decision letter ready bubble is still present.  Approved by QA.
